provisioning/ubuntu-provision.sh

```diff
@@ -62,7 +62,7 @@ function clean_apt() {
   apt-get autoremove --purge -y
 
   ## Ensure the machine is up-to-date
-  apt-get update
+  apt-get update --quiet
```

Comment:
Maybe create an `apt_update` function?
WDYT @dduportal @smerle33 ?
(nit)

Reply:
One line to factorize one line seems weird